graph/export.go

```diff
@@ -96,10 +99,10 @@ func (s *TagStore) CmdImageExport(job *engine.Job) error {
 	}
 	defer fs.Close()
 
-	if _, err := io.Copy(job.Stdout, fs); err != nil {
+	if _, err := io.Copy(imageExportConfig.Outstream, fs); err != nil {
 		return err
 	}
-	logrus.Debugf("End export job: %s", job.Name)
+	logrus.Debugf("End export job: image_export")
```

Comment:
I'll totally remove this line since it's not a job anymore, or maybe write another message